src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Count.cs

```diff
@@ -22,8 +22,6 @@ internal sealed class CountFunction : FunctionWithTableInput
     {
         public override bool IsSelfContained => true;
 
-        public override bool SupportsParamCoercion => false;
```

Comment:
>SupportsParamCoercion [](http://example.com/codeflow?start=29&length=21)

The base class (FunctionWithTableInput) already forbids coercions for the first argument (and this function only has one)